docs/services/camera_sim.rst

```diff
@@ -1,14 +1,59 @@
 Camera Simulator
 ================
+This service operates a simulated camera. This service is meant to mimic, but does not actually
+control, a hardware camera.
+
+When applicable, all services have a corresponding simulated service to be able to test control
+software before commanding actual hardware devices.
 
 Configuration
 -------------
+.. code-block:: YAML
+
+    camera:
+      service_type: zwo_camera
+      simulated_service_type: camera_sim
+      interface: hicat_camera
+      requires_safety: false
+
+      device_name: ZWO ASI178MM
+      device_id: 4
+      well_depth_percentage_target: 0.65
```

Comment:
Could you put in a blank space after this line, so indicate the difference between the keys required by the camera_sim service (exposure_time, gain, etc.) and the keys that are used by the hardware service?

Reply:
done, also added comments to better make that distinction.